testing-scripts/sqoop/test_sqoop.py

```diff
@@ -0,0 +1,102 @@
+
+"""
+This module provides testing functionality of the Sqoop Init Action.
+
+Test logic:
+1. Fill some data into cloud SQL using cloud-sql-proxy.
+2. Use sqoop import job to get data from cloud SQL and put it into hdfs.
+
+Note:
+    Hive init action was added to provide additional libs to sqoop.
+    In that example mysql connector will be added to sqoop's classpath.
+"""
+
+import unittest
+import random
+
+from parameterized import parameterized
+
+from dataproc_test_case import DataprocTestCase
+
+
+class SqoopTestCase(DataprocTestCase):
```

Comment:
Let's leave this one out, and instead add it to #269 .

Reply:
ack